TypeCobol.Analysis.Test/TypeCobol.Analysis.Test.csproj

```diff
@@ -20,6 +20,9 @@
     <Reference Include="Microsoft.VisualStudio.QualityTools.UnitTestFramework, Version=10.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, processorArchitecture=MSIL">
       <Private>False</Private>
     </Reference>
+    <Reference Include="Antlr4.Runtime, Version=4.6.0.0, Culture=neutral, PublicKeyToken=09abb75b9ed49849, processorArchitecture=MSIL">
+      <HintPath>..\packages\Antlr4.Runtime.4.6.6\lib\net45\Antlr4.Runtime.dll</HintPath>
```

Comment:
Is there a problem with the versions? One is 4.6.0.0, the other is 4.6.6.
And there is `net45` in the path, while is it written `targetFramework="net47"` in the file packages.config.

Reply:
No it's fine, actually i've copied the reference from `TypeCobol.Analysis` project file.
- the assembly version is `4.6.0.0` but the NuGet package version is `4.6.6`
- we are targeting .NET Framework 4.7 but Antlr provides assemblies up to `net45` so we are using the most up-to-date available DLL (which is compatible with our own target).